lib/actions/device.coffee

```diff
@@ -384,8 +384,8 @@ exports.init =
 				if process.env.DEBUG
 					console.log(results.config)
 
-				bar = new visuals.widgets.Progress('Downloading Device OS')
-				spinner = new visuals.widgets.Spinner('Downloading Device OS (size unknown)')
+				bar = new visuals.Progress('Downloading Device OS')
```

Comment:
Did you test that progress bar information is displayed correctly (I think I introduced breaking changes there)?